src/components/card/video-card/VideoCard.jsx

```diff
@@ -15,9 +18,18 @@ export const VideoCard = ({ video, loading }) => {
     isInWatchLater,
     isLiked,
     videoYTId,
+    _id,
   } = video;
   const optionsList = getOptionsList(isInWatchLater, isLiked);
 
+  const handleOverlayPlayClick = () => {
+    navigate(`${ROUTE_WATCH_VIDEO}/${_id}`);
+  };
+
+  const handleKeyDownOnOverlayPlay = (event) => {
+    if (event.key === "Enter") handleOverlayPlayClick();
+  };
+
   return loading ? (
     <div
       className={`card card-vertical shadow-unset ${styles.videoCard_loading} ${styles.videoCard}`}
```

Comment:
you can use classnames npm package if you want to. It will help write many no of classes separating by ,

Reply:
Thanks for suggesting, I will check about classnames.